tests/testthat/test-stacks.R

```diff
@@ -0,0 +1,71 @@
+skip_if_not_installed("stacks")
+skip_if_not_installed("ranger")
```

Comment:
Unless this is a needed vetiver Suggests, we should be able to remove this line! The `class_res_rf` object is the stacks example object with a ranger dependency.

```suggestion

```